src/components/app/MenuButtons.js

```diff
@@ -508,12 +500,7 @@ class ProfileSharingCompositeButton extends React.PureComponent<
 
   render() {
     const { state, uploadProgress, error, shortUrl } = this.state;
-    const { profile, symbolicationStatus, profileSharingStatus } = this.props;
-
-    const shareLabel =
-      symbolicationStatus === 'DONE'
-        ? 'Share...'
-        : 'Sharing will be enabled once symbolication is complete';
```

Comment:
Note: this behavior was added so that we couldn't upload an unsymbolicated profile. Now it's not necessary anymore.

Reply:
As a follow-up I would like to make it possible to re-upload a profile after symbolication, if symbolication has been done. I didn't want to overcharge this PR with this.

I removed this mainly because of previous discussions with Florian, where capturing and symbolicating profiles on slow computers is painful. So with this change at least he'd be able to upload a profile. But maybe we'd want to address this differently?

Reply:
My current position is to check this in, but we can very easily revert just this commit in case we see there are problems.